demisto_sdk/commands/common/hook_validations/graph_validator.py

```diff
@@ -0,0 +1,281 @@
+
+        is_valid = True
+        paths_with_invalid_marketplaces = (
+            self.graph.find_uses_paths_with_invalid_marketplaces(file_paths)
+        )
+
+        for query_result in paths_with_invalid_marketplaces:
+            content_name = query_result.name
+            relationship_data = query_result.uses
+            marketplaces = query_result.marketplaces
+            content_items = [
+                relationship.target.name
+                if relationship.target.name
+                else relationship.target.object_id
+                for relationship in relationship_data
+            ]
+            file_path = query_result.path
+            self.ignored_errors = ValidateManager.get_error_ignore_list(
+                get_pack_name(file_path)
+            )
+            error_message, error_code = Errors.uses_with_invalid_marketplaces(
+                content_name, marketplaces, content_items
+            )
+            if self.handle_error(error_message, error_code, file_path, warning=True):
+                is_valid = False
+
+        return is_valid
+
+    @error_codes("GR102")
+    def is_file_using_unknown_content(self, file_paths=[]):
+        """
+        Validate that there are no usage of unknown content items
+        """
+        from demisto_sdk.commands.validate.validate_manager import ValidateManager
+
+        is_valid = True
+        query_results = self.graph.get_unknown_content_uses(file_paths)
+
+        if query_results:
+            for query_result in query_results:
+                content_name = query_result.name
+                relationship_data = query_result.uses
+                unknown_content_names = [
+                    relationship.target.identifier for relationship in relationship_data
+                ]
+                file_path = query_result.path
+                self.ignored_errors = ValidateManager.get_error_ignore_list(
+                    get_pack_name(file_path)
+                )
+                error_message, error_code = Errors.using_unknown_content(
+                    content_name, unknown_content_names
+                )
+                if self.handle_error(
+                    error_message, error_code, file_path, warning=True
+                ):
+                    is_valid = False
+
+        return is_valid
+
+    @error_codes("GR103")
+    def is_file_display_name_already_exists(self, file_paths=[]):
+        """
+        Validate that there are no duplicate display names in the repo
+        """
+        is_valid = True
+        pack_names = []
+        if file_paths:
+            pack_names = get_pack_display_name_from_file_in_pack(file_paths)
+        query_results = self.graph.get_duplicate_pack_display_name(pack_names)
+
+        if query_results:
+            for query_result in query_results:
+                content_id = query_result[0]
+                duplicate_names_id = query_result[1]
```

Comment:
```suggestion
            for content_id, duplicate_names_id in query_results:
```
assuming these are all two-tuples